context/app/static/js/pages/Template/style.ts

```diff
@@ -0,0 +1,24 @@
+import AccordionSummary from '@mui/material/AccordionSummary';
+import { styled } from '@mui/material/styles';
+
+const StyledAccordionSummary = styled(AccordionSummary)(({ theme }) => {
```

Comment:
These styles seem pretty similar to the ones for the Processed Dataset accordion: https://github.com/hubmapconsortium/portal-ui/blob/25b9fea3fca9f6bc8d789b6c8851f7e44779e213/context/app/static/js/components/detailPage/ProcessedData/ProcessedDataset/styles.ts

Maybe there's some deduplication we could do? It doesn't have to be in this PR, but we should file a ticket otherwise.

Reply:
Oh nice! Good call.